medicines/doc-index-updater/src/create_manager/models.rs

```diff
@@ -128,12 +223,12 @@ mod test {
         let expected_doc_type = "Spc".to_string();
         let expected_title = "Paracetamol Plus PL 12345/6789".to_string();
         let expected_author = "JRR Tolkien".to_string();
-        let expected_product_name = "[\"Effective product 1\",\"Effective product 2\"]".to_string();
-        let expected_substance_name = "[\"Paracetamol\",\"Caffeine\"]".to_string();
+        let expected_product_name = "[\"EFFECTIVE PRODUCT 1\",\"EFFECTIVE PRODUCT 2\"]".to_string();
+        let expected_substance_name = "[\"PARACETAMOL\",\"CAFFEINE\"]".to_string();
```

Comment:
Just to double-check—this case conversion is something we're expected to do to normalise unpredictable data, yes?

Reply:
yeah, I'm doing it here so BLOB matches INDEX and the DRUG INDEX/SUBSTANCE pages work as they have done previously
@TimboTambo and @m-doughty as importer experts - does this seem right to you?

Reply:
Seems right to me.